modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/xstream/configurator/DynamicDataMappingXStreamConfigurator.java

```diff
@@ -71,6 +72,7 @@ protected void activate() {
 				com.liferay.dynamic.data.mapping.model.LocalizedValue.class),
 			new XStreamType(
 				com.liferay.dynamic.data.mapping.storage.DDMFormValues.class),
+			new XStreamType(DDMFormRule.class),
 			new XStreamType(DDMFormField.class),
 			new XStreamType(DDMFormFieldOptions.class),
 			new XStreamType(DDMFormFieldValidation.class),
```

Comment:
Hi @jorgediaz-lr. 
Just one question, is it possible to sort this alphabetically? `DDMFormRule` after `DDMFormFieldValidation`. Thanks!

Reply:
Hi @carolmariaabb 

I have updated the PR with the correct order see new commit https://github.com/liferay-forms/liferay-portal/pull/1155/commits/5e93fa8758a3a78ca2b284a5e92ae993edda57cb

Reply:
Thanks @jorgediaz-lr!